community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/util.py

```diff
@@ -1923,26 +1939,62 @@ def nodeset_map(self, hostnames: list):
             nodeset_map[self.node_nodeset_name(node)].append(node)
         return nodeset_map
 
+    @lru_cache
+    def get_jobs(self) -> List[Job]:
```

Comment:
`job` and `get_jobs` should return the same level of details.
Please create an internal function that can parse the the output format of `scontrol` and reuse it.

Reply:
done.